templates/blog/comment_create.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h1>Додати коментар до поста</h1>
+
+  <form method="post" action="{% url 'blog:create' post.id %}">
```

Comment:
The action URL seems to be incorrect. Ensure that the URL name 'blog:create' matches the URL pattern defined in your Django URL configuration for creating a comment. It might need to be something like 'blog:comment_create' or similar, depending on your URL patterns.